src/containers/Tenant/QueryEditor/QueryResult/QueryResult.js

```diff
@@ -94,31 +94,33 @@ function QueryResult(props) {
     };
 
     const renderIssues = () => {
-        const error = props.error?.data;
+        const error = props.error;
 
-        const hasIssues = error?.issues && Array.isArray(error.issues);
+        const hasIssues = error?.data?.issues && Array.isArray(error.data.issues);
 
         return hasIssues ? (
             <React.Fragment>
-                <ResultIssues data={error} />
+                <ResultIssues data={error.data} />
                 {isFullscreen && (
                     <Fullscreen>
                         <div className={b('result', {fullscreen: true})}>
-                            <ResultIssues data={error} />
+                            <ResultIssues data={error.data} />
                         </div>
                     </Fullscreen>
                 )}
             </React.Fragment>
         ) : (
-            <span>{error?.data ?? error}</span>
+            <div className={b('error')}>
+                {error?.message || error?.data || error}
```

Comment:
Can error data be an object? Or error itself? Won't it fail with "Objects are not valid as a React child"? Maybe it's better to use `prepareQueryError` as it seems to fit here. Or at least parse it to string